### ISSUE TITLE
Linting tests failing 

### ISSUE THREAD

**Issue by ggabernet** [nf-core/bcellmagic#34]
Hey there are a couple of linting tests failing still in Travis, the rest passed! 

INFO: ===========
 LINTING RESULTS
=================
  82 tests passed   2 tests had warnings   2 tests failed
WARNING: Test Warnings:
  http://nf-co.re/errors#4: Config variable not found: params.reads
  http://nf-co.re/errors#4: Config variable not found: params.singleEnd
ERROR: Test Failures:
  http://nf-co.re/errors#8: Conda environment name is incorrect (nf-core-bcellmagic-1.0, should be nf-core-bcellmagic-1.0dev)
  http://nf-co.re/errors#8: Conda dependency did not have pinned version number: bioconda::biopython
ERROR: Sorry, some tests failed - exiting with a non-zero error code...

**Comment by subwaystation** [comment]
As far as I got from http://nf-co.re/errors#4, we can ignore the warnings.
Conda environment name is incorrect -> I will fix this.
Conda dependency did not have pinned version number: bioconda::biopython -> As far as I got it from the linting discussion, the linting tool can not resolve this on its own. I will add a specific biopython version. I will also built the container again, to verify, that it is working.
